### ISSUE TITLE
gapfill a modelseed model

### ISSUE THREAD

**Issue by hariszaf** [canc1993/cheshire-gapfilling#3]
Hi @liaochen1988 . 
This is a general question. 
In order to gap fill a modelseed model, one would need an analogous file with the `bigg_universe.xml` of yours; is that right ? 

On top of that, and just to make sure, the draft reconstruction needs to ensure growth on its own, already before running `cheshire`, correct?

Thank you very much in advance.

**Comment by liaochen1988** [comment]
Yes, exactly. A universal reaction pool from modelseed is needed for gap filling modelseed models.

Regarding draft reconstructions, yes again, the draft model needs to grow before running Cheshire. Cheshire does gapfill growth on its own.
